### PR TITLE
always return http as type for toRpcHttp

### PR DESCRIPTION
**Pull request by christopheranderson**
fixes https://github.com/Azure/azure-functions-nodejs-worker/issues/52

tests will be added in functions-host repo (since they are still there...)

### CODE REVIEW

File: src/Converters.ts [hunk 1]
```diff
@@ -16,15 +16,11 @@ export function fromRpcHttp(rpcHttp: rpc.RpcHttp$Properties) {
 }
 
 export function toRpcHttp(inputMessage): rpc.TypedData$Properties {
-  if (inputMessage.body !== undefined) {
     let httpMessage: rpc.RpcHttp$Properties = inputMessage;
```
Comment (pragnagopa) [inline src/Converters.ts]: I think we made an assumption here that if the body is present rest of the HttpMessage properties would be available.  It's possbile inputMessage.statusCode might be missing. Do we have tests for this?

Comment (christopheranderson) [reply src/Converters.ts]: We do in the main repo. I'll have another PR on azure-functions-host which will add all the HTTP tests back in.

Comment (christopheranderson) [reply src/Converters.ts]: I've already run them and they pass. :) Need to make it so we can run those tests on PRs, though.

Comment (christopheranderson) [reply src/Converters.ts]: Also, it's ok for status to be undefined/null/etc. It doesn't break today if you don't have status set. It just gets the default from the WebHost.

**Pull request merged**